lib/install.js

```diff
@@ -57,6 +59,8 @@ function install(argv, callback) {
     pmSeedEnv: {},
     _touched: [], // not a real config, used for recording paths to chown
     dirs: [],
+    useMemoryDb: false,
+    dbUpdate: false,
```

Comment:
dbUpdate seems unused... I think checkAndUpgrade always occurs, even if the mem db is being used?